profiles/BSMPT/macos-debug

```diff
@@ -0,0 +1,8 @@
+[settings]
+arch=x86_64
+build_type=Debug
+compiler=apple-clang
+compiler.cppstd=gnu17
+compiler.libcxx=libc++
+compiler.version=13
```

Comment:
Need to increase version here?

Reply:
I just ran detect on the macos runner as I don't have a mac system.

Reply:
Looking at the [docs](https://docs.conan.io/2/reference/config_files/profiles.html), `compiler.version=14` seems to exist at least, but there's probably no immediate need for us to increase this. If this is what the macos runner has, that sounds fine.